packages/x-data-grid/src/components/containers/GridRootStyles.ts

```diff
@@ -689,13 +689,6 @@ export const GridRootStyles = styled('div', {
     [`& .${c['filler--borderTop']}`]: {
       borderTop: '1px solid var(--DataGrid-rowBorderColor)',
     },
-
-    /* Hide grid rows and vertical scrollbar when skeleton overlay is visible */
-    [`& .${c['main--hasSkeletonLoadingOverlay']}`]: {
-      [`& .${c.virtualScrollerContent}, & .${c['scrollbar--vertical']}, & .${c.pinnedRows}`]: {
-        display: 'none',
```

Comment:
I tried doing this instead:
```
visibility: 'hidden',
position: 'fixed',
```
It seems to work, but I'm unsure if this would work in every scenario.
What do you think?

Reply:
It does seem to work - I thought we tried this the other day 😅. I will do some more thorough testing to double check different scenarios.

Reply:
This seems to be the ideal combination here. The virtual scroller content retains it's height with `visibility: hidden`, fixing the issue with nested data grid. And `position: fixed` takes the element out of the flow. Will make the change, thanks @cherniavskii for taking a look.

Reply:
Great! 🎉